pkg/virt-operator/resource/generate/components/daemonsets.go

```diff
@@ -323,7 +323,7 @@ func NewHandlerDaemonSet(namespace, repository, imagePrefix, version, launcherVe
 			corev1.ResourceCPU: resource.MustParse("10m"),
 			// 325Mi - base memory request
```

Comment:
As is commonplace, comments rot. Can we avoid them with
```suggestion
			baseMem := "325Mi"
			containerDiskChecksumBuffer := "32Mi"
			resource.MustParse(baseMem).Add(resource.MustParse(containerDiskChecksumBuffer))
			
```
?

Reply:
I think I will just use the full value, as the next future will be like `if fetureGate is enabled += 34Mi` do we want to count the memeory usage of each function?